packages/office-addin-debugging/src/commands.ts

```diff
@@ -1,121 +1,123 @@
-        const enableLiveReload: boolean = (command.liveReload === true);
-        const enableSideload: boolean = (command.sideload !== false); // enable if true or undefined; only disable if false
-        const openDevTools: boolean = (command.devTools === true);
-        const packager: string | undefined = command.packager || process.env.npm_package_scripts_packager;
-        const packagerHost: string | undefined = command.PackagerHost || process.env.npm_package_config_packager_host;
-        const packagerPort: string | undefined = command.PackagerPort || process.env.npm_package_config_packager_port;
-        const sourceBundleUrlComponents = new devSettings.SourceBundleUrlComponents(
-            command.sourceBundleUrlHost, command.sourceBundleUrlPort,
-            command.sourceBundleUrlPath, command.sourceBundleUrlExtension);
-
-        if (appPlatformToDebug === undefined) {
-            throw new ExpectedError("Please specify the platform to debug.");
-        }
-
-        if (appTypeToDebug === undefined) {
-            throw new ExpectedError("Please specify the application type to debug.");
-        }
-
-        if (appPlatformToDebug === Platform.Android || appPlatformToDebug === Platform.iOS) {
-            throw new ExpectedError(`Platform type ${appPlatformToDebug} not currently supported for debugging`);
-        }
-
-        if (!manifestPath) {
-            manifestPath = determineManifestPath(appPlatformToDebug, dev);
-        }
-
-
-        await startDebugging(manifestPath, {
-            appType: appTypeToDebug,
-            app,
-            debuggingMethod,
-            sourceBundleUrlComponents,
-            devServerCommandLine: devServer,
-            devServerPort,
-            packagerCommandLine: packager,
-            packagerHost,
-            packagerPort,
-            enableDebugging,
-            enableLiveReload,
-            enableSideload,
-            openDevTools,
-            document
-        });
-
-        usageDataObject.reportSuccess("start");
-    } catch (err) {
-        usageDataObject.reportException("start", err);
-        logErrorMessage(`Unable to start debugging.\n${err}`);
-    }
-}
-
-export async function stop(manifestPath: string, platform: string | undefined, command: commander.Command) {
-    try {
-        const appPlatformToDebug: Platform | undefined = parsePlatform(platform || process.env.npm_package_config_app_plaform_to_debug || Platform.Win32);
-        const dev: boolean = command.prod ? false : true;
-
-        if (manifestPath === "" && appPlatformToDebug !== undefined) {
-            manifestPath = determineManifestPath(appPlatformToDebug, dev);
-        }
-
-        await stopDebugging(manifestPath);
-        usageDataObject.reportSuccess("stop");
-    } catch (err) {
-        usageDataObject.reportException("stop", err);
-        logErrorMessage(`Unable to stop debugging.\n${err}`);
-    }
-}
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
There are whitespace issues that cause the diff to show changes on every line. Please update so I can actually tell what changed here.

Reply:
I don't know why these files were created. I didn't even ran ```lint```, just the regular ```install``` and ```build``` and those changes were created.
What could it be?